doc/classes/String.xml

```diff
@@ -549,78 +617,79 @@
 			<param index="1" name="base" type="int" default="10" />
 			<param index="2" name="capitalize_hex" type="bool" default="false" />
 			<description>
-				Converts a unsigned [int] to a string representation of a number.
+				Converts the given unsigned [int] to a string representation, with the given [param base].
+				By default, [param base] is set to decimal ([code]10[/code]). Other common bases in programming include binary ([code]2[/code]), [url=https://en.wikipedia.org/wiki/Octal]octal[/url] ([code]8[/code]), hexadecimal ([code]16[/code]).
+				If [param capitalize_hex] is [code]true[/code], number representations after base 10 are returned in uppercase.
 			</description>
 		</method>
 		<method name="pad_decimals" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="digits" type="int" />
 			<description>
-				Formats a number to have an exact number of [param digits] after the decimal point.
+				Formats the string representing a number to have an exact number of [param digits] [i]after[/i] the decimal point.
 			</description>
 		</method>
 		<method name="pad_zeros" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="digits" type="int" />
 			<description>
-				Formats a number to have an exact number of [param digits] before the decimal point.
+				Formats the string representing a number to have an exact number of [param digits] [i]before[/i] the decimal point.
 			</description>
 		</method>
 		<method name="path_join" qualifiers="const">
 			<return type="String" />
 			<param index="0" name="file" type="String" />
 			<description>
-				If the string is a path, this concatenates [param file] at the end of the string as a subpath. E.g. [code]"this/is".path_join("path") == "this/is/path"[/code].
+				If the string is a file path, this concatenates [param file] at the end of the string, as a subpath. E.g. [code]"this/is".path_join("path") == "this/is/path"[/code].
```

Comment:
And if it's not a valid file path? Also remove "this" before "concatenates". Wrong comma before "as" also.

Reply:
The correct wording for it would be "Assuming the string is a file path", actually. There's no way the method fails. But beginning with "assuming"  is a methodology I've never seen, so it may need some other rewording.

Reply:
Rewrote that mess entirely at this point.